### ISSUE TITLE
getPeers plugin deprecation warning

### ISSUE THREAD

**Issue by tynes** [bpanel-org/bpanel#24]
To render the locations of the peers on google maps in the `getPeers` plugin, `freegeoip.net` is used.

When the site is served with TLS, the requests fail because `http://freegeoip.net` is hard coded in, that needs to be replaced with `https://freegeoip.net`

The service is also going to be deprecated soon

```
$ curl -s http://freegeoip.net/json/35.188.2.192 | jq .
{
  "__deprecation_message__": "This API endpoint is deprecated and will stop working on July 1st, 2018. For more information please visit: https://github.com/apilayer/freegeoip#readme",
  "ip": "35.188.2.192",
  "country_code": "US",
  "country_name": "United States",
  "region_code": "CA",
  "region_name": "California",
  "city": "Mountain View",
  "zip_code": "94043",
  "time_zone": "America/Los_Angeles",
  "latitude": 37.4192,
  "longitude": -122.0574,
  "metro_code": 807
}
```

**Comment by bucko13** [comment]
Good catch. Looks like they've moved to a more professional service since I found them a couple months ago. When this widget becomes more robust, it might make sense to allow for users to add their own API key with this service (makes me think that a plugin setting UI has even more utility as well), but for now since this is just a naive implementation, I've just moved to a different API. See here: https://github.com/bpanel-org/peers-widget/commit/6b4f1d48aae2b9fedeeb4ff61b89057288662cbb (also moved the plugin to the bpanel-org)
